docs/src/main/paradox/docs/delta/api/search-api.md

```diff
@@ -0,0 +1,39 @@
+# Global search
+
+Nexus provides global search functionality across all projects through the @link:[search plugin](https://github.com/BlueBrain/nexus/tree/master/delta/plugins/search/src){ open=new }.
+
+The plugin is disabled by default and requires following configuration:
+
+- `plugins.search.enabled=true` - enables the plugin. Enabling the plugin will trigger creating a `CompositeView` in each project.
+- `plugins.search.fields={pathToFile}` - specifies which fields are available for searching and/or aggregations. This also serves as configuration returned by the `/search/config` endpoint. Example can be found @link:[here](https://github.com/BlueBrain/nexus/blob/master/tests/docker/config/fields.json){ open=new }.
```

Comment:
We should add that this current file is needed to use global search with Fusion.
If they don't use Fusion, this could be optional

It is ok for now as we will have to update it anyway in this current sprint but as it is something custom, we should document more the different values and options

Also add a link to its counterpart in Fusion doc when it gets written